### PR TITLE
No legs? No walkies

### LINKED ISSUE #8007: You can walk without legs
### Environment
- Game Version: 
- OS Version: 

### Detailed description of bug
You can still walk (although with a bit less speed) without both legs instead of crawling.

### Steps to reproduce
1. Cut off both of your legs.
2. Walk.

### PR DESCRIPTION
**Pull request by MaxIsJoe**
Fixes #8007

CL: [Improvement] - Players who have no legs will be forced to crawl on the ground and cannot stand up until they get a new pair.

### CODE REVIEW

File: UnityProject/Assets/Scripts/Items/Implants/Limb.cs [hunk 1]
```diff
@@ -34,12 +34,21 @@ public class Limb : BodyPartFunctionality, IMovementEffect
 
 		private PlayerHealthV2 playerHealth;
 
+		public bool IsLeg = false;
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Items/Implants/Limb.cs:37]: isn't the leg a class? If it is, you can test it for the type instead which is a lot cleaner than adding bools only one type of limb will use

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Items/Implants/Limb.cs:37]: A leg isn't a class, legs and arms are limbs and there is no way to tell them apart currently.

Comment (AtnerNT) [reply UnityProject/Assets/Scripts/Items/Implants/Limb.cs:37]: Is there a leg item trait you can use?

Comment (corp-0) [reply UnityProject/Assets/Scripts/Items/Implants/Limb.cs:37]: how do we store legs stats then? Like movement speed they give

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Items/Implants/Limb.cs:37]: just like how `IsLeg` is done :) right above that is a bunch of variables for both the arms and legs and stuff grabs it from here.
A bit ugly innit? Tho this PR isn't really meant to clean that up but I left a little reminder for myself in a form of an issue (#8987) to clean it up later.

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Items/Implants/Limb.cs:37]: hewwo?

Comment (corp-0) [reply UnityProject/Assets/Scripts/Items/Implants/Limb.cs:37]: I think I can allow this as long as we pinky promise to do #8987

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Items/Implants/Limb.cs:37]: Pinky promise

**Review by corp-0 — APPROVED** [review]

**Pull request merged**